### ISSUE TITLE
Inferencing with .pb file ?

### ISSUE THREAD

**Issue by ShreshthSaxena** [tucan9389/PoseEstimation-CoreML#24]
The model folder contains protobuf, coreml and tflite files for both hourglass and cpm.
Should I expect the model performance to be similar in all three files?
I yes, is there an inference code to test the .pb file (or any other) in Colab and visualise the results ?

**Comment by tucan9389** [comment]
> The model folder contains protobuf, coreml and tflite files for both hourglass and cpm.
> Should I expect the model performance to be similar in all three files?
> I yes, is there an inference code to test the .pb file (or any other) in Colab and visualise the results ?

@ShreshthSaxena 
No, the performance could be different between them and there is no inference code to test as far as I know. 
Because the model interpreters are still in development now. If you measure the performances, please share the result with contribution. I really want to check the point, but too busy these days 😢
